packages/checkbox/checkbox.test.mts

```diff
@@ -34,7 +34,7 @@ describe('checkbox prompt', () => {
        ◯ 5
        ◯ 6
        ◯ 7
-      (Move up and down to reveal more choices)"
+      (Use arrow keys to reveal more choices)"
```

Comment:
I made this change on the master branch to reduce the amount of diff noise generated on your PR. Do you mind re-merging master on top of your changes?

Reply:
That's great, thank you so much! I'll do that.